### PR TITLE
Add changelog for 1.2 release

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
 /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.2.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+
+   * [v1.2.0](#v120)
+      * [Downloads for v1.2.0](#downloads-for-v110)
```
Comment (kevin-wangzefeng) [inline CHANGELOG-1.2.md]:
```suggestion
      * [Downloads for v1.2.0](#downloads-for-v120)
```

File: CHANGELOG-1.2.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+         * [KubeEdge Binaries](#kubeedge-binaries)
+         * [Installer Binaries](#installer-binaries)
+         * [EdgeSite Binaries](#edgesite-binaries)
+      * [KubeEdge v1.2 Release Notes](#kubeedge-v11-release-notes)
```
Comment (kevin-wangzefeng) [inline CHANGELOG-1.2.md]:
```suggestion
      * [KubeEdge v1.2 Release Notes](#kubeedge-v12-release-notes)
```

File: CHANGELOG-1.2.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+# KubeEdge v1.2 Release Notes
+
+## 1.2 What's New
+
+**Reliable message delivery from cloud to edge**
+
+This feature improved the reliable message delivery mechanism from cloud to edge. If cloudcore or edgecore
+being restarted or offline for a while, it can ensure that the messages sent to the edge are not lost, and 
+ensure inconsistency between cloud and edge.
```
Comment (kevin-wangzefeng) [inline CHANGELOG-1.2.md]:
```suggestion
avoid inconsistency between cloud and edge.
```

File: CHANGELOG-1.2.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+([#1343](https://github.com/kubeedge/kubeedge/pull/1343), [@kevin-wangzefeng](https://github.com/arcanique), [@fisherxu](https://github.com/fisherxu), [@SpaghettiAndSalmon](https://github.com/SpaghettiAndSalmon))
```
Comment (kevin-wangzefeng) [inline CHANGELOG-1.2.md]:
```suggestion
([#1343](https://github.com/kubeedge/kubeedge/pull/1343), [@kevin-wangzefeng](https://github.com/), [@fisherxu](https://github.com/fisherxu), [@SpaghettiAndSalmon](https://github.com/SpaghettiAndSalmon))
```

**Comment by fisherxu** [comment]
> minor comments

Done.

**Pull request merged**